content/en/docs/instrumentation/js/libraries.md

```diff
@@ -14,253 +51,221 @@ For example,
-packages to instrument inbound and outbound HTTP traffic.
-
-```shell
-npm install --save @opentelemetry/instrumentation-http @opentelemetry/instrumentation-express
-```
-
-And then register each instrumentation library:
+To only load individual instrumentation libraries, replace
+`[getNodeAutoInstrumentations()]` with the list of those you need:
 
 <!-- markdownlint-disable -->
 <!-- prettier-ignore-start -->
 {{< tabpane langEqualsHeader=true >}}
 
 {{< tab TypeScript >}}
-/* tracing.ts */
-
-// Import dependencies
+/*instrumentation.ts*/
+...
 import { HttpInstrumentation } from "@opentelemetry/instrumentation-http";
 import { ExpressInstrumentation } from "@opentelemetry/instrumentation-express";
-import opentelemetry from "@opentelemetry/api";
-import { Resource } from "@opentelemetry/resources";
-import { SemanticResourceAttributes } from "@opentelemetry/semantic-conventions";
-import { NodeTracerProvider } from "@opentelemetry/sdk-trace-node";
-import { registerInstrumentations } from "@opentelemetry/instrumentation";
-import { ConsoleSpanExporter, BatchSpanProcessor } from "@opentelemetry/sdk-trace-base";
-
-// This registers all instrumentation packages
-registerInstrumentations({
+
+const sdk = new NodeSDK({
+  ...
   instrumentations: [
     // Express instrumentation expects HTTP layer to be instrumented
     new HttpInstrumentation(),
     new ExpressInstrumentation(),
-  ],
+  ]
 });
 
-const resource =
-  Resource.default().merge(
-    new Resource({
-      [SemanticResourceAttributes.SERVICE_NAME]: "service-name-here",
-      [SemanticResourceAttributes.SERVICE_VERSION]: "0.1.0",
-    })
-  );
+sdk.start()
 
-const provider = new NodeTracerProvider({
-    resource: resource,
-});
-const exporter = new ConsoleSpanExporter();
-const processor = new BatchSpanProcessor(exporter);
-provider.addSpanProcessor(processor);
-
-provider.register();
 {{< /tab >}}
 
 {{< tab JavaScript >}}
-/* tracing.js */
-
-// Require dependencies
+/*instrumentation.js*/
 const { HttpInstrumentation } = require("@opentelemetry/instrumentation-http");
 const { ExpressInstrumentation } = require("@opentelemetry/instrumentation-express");
-const opentelemetry = require("@opentelemetry/api");
-const { Resource } = require("@opentelemetry/resources");
-const { SemanticResourceAttributes } = require("@opentelemetry/semantic-conventions");
-const { NodeTracerProvider } = require("@opentelemetry/sdk-trace-node");
-const { registerInstrumentations } = require("@opentelemetry/instrumentation");
-const { ConsoleSpanExporter, BatchSpanProcessor } = require("@opentelemetry/sdk-trace-base");
 
-// This registers all instrumentation packages
-registerInstrumentations({
-  instrumentations: [
-    // Express instrumentation expects HTTP layer to be instrumented
-    new HttpInstrumentation(),
-    new ExpressInstrumentation(),
-  ],
+const sdk = new NodeSDK({
+  ...
+  // This registers all instrumentation packages
+  instrumentations: [getNodeAutoInstrumentations()]
```

Comment:
Looks like these two sections got switched on the JavaScript tab - `getNodeAutoInstrumentations()` should be up on line 124, and http/express should be here.

Reply:
fixed :-)